CHANGELOG.md

```diff
@@ -1,10 +1,78 @@
 # MultiQC Version History
 
+## [MultiQC v1.25](https://github.com/MultiQC/MultiQC/releases/tag/v1.25) - 2024-09-16
+
+### Highlights - sample grouping
+
+New feature: grouping samples in the General Statistics table.
+
+<img width="400" src="docs/images/changelog/v1.25-grouping-samples.png">
+
+Some modules - prominently FastQC - may produce multiple results per sample, e.g. for the forward and the reverse reads. To group such results in the table together, a new [configuration option](docs/core/reports/customisation.md#sample-grouping) is introduced.
+
+It's opt-in, but will be on by default in the future releases.
+
+Because MultiQC needs to know how to merge each column (sum, average, etc), each module must implement it independenly. Currently it's supported by the FastQC and Cutadapt modules. To add it into other modules, follow the [development documentation](docs/core/development/modules.md#grouping-samples).
```

Comment:
```suggestion
Because MultiQC needs to know how to merge each column (sum, average, etc), each module must implement it independently. Currently it's supported by the FastQC and Cutadapt modules. If you'd like support added to another module, please let us know in a GitHub issue. Details of how to add it into module code can be found in the [moduel development documentation](https://multiqc.info/docs/development/modules/#grouping-samples).
```